### PR TITLE
fix: add margin left to prevent overflow of nav item

### LINKED ISSUE #3889: Bug: outline cut off for dashboard tab when it receives focus
@nickytonline not a blocker, but the dashboard outline gets cut off (I'm on chrome)

image

That's not related to the fix, but we can address that in a separate issue. Probably a good first issue for a11y for someone.

_Originally posted by @BekahHW in https://github.com/open-sauced/app/issues/3888#issuecomment-2276088006_

The page in question is https://beta.app.opensauced.pizza/explore/topic/typescript?redirectedFrom=%2F, but I imagine it happens anywhere that the tab component is used.

### PR DESCRIPTION
**Pull request by Dun-sin**
## Description
This PR adds the class `ml-2` to the nav component to prevent the overflow

## Related Tickets & Documents

fix #3889

## Mobile & Desktop Screenshots/Recordings
before:
image

after:

https://github.com/user-attachments/assets/16e40c58-9fd8-41cb-8c9b-a9697148914d

## Steps to QA

1. Go to this URL → https://beta.app.opensauced.pizza/explore/topic/typescript?redirectedFrom=%2F
2. Use the tab to navigate to the nav options e.g (dashboard, contributors etc)

## Tier (staff will fill in)

- [ ] Tier 1
- [ ] Tier 2
- [ ] Tier 3
- [ ] Tier 4

### CODE REVIEW

File: components/organisms/ToolList/nav.tsx [hunk 1]
```diff
@@ -1,9 +1,7 @@
 import React from "react";
-
 import { useRouter } from "next/router";
```
Comment (Dun-sin) [inline components/organisms/ToolList/nav.tsx:2]:
```suggestion

import { useRouter } from "next/router";
```

File: components/organisms/ToolList/nav.tsx [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
-import { setQueryParams } from "lib/utils/query-params";
-
 import ComponentDateFilter from "components/molecules/ComponentDateFilter/component-date-filter";
+import { setQueryParams } from "lib/utils/query-params";
 import NavItem from "./nav-item";
 
 type ToolListArray = {
```
Comment (Dun-sin) [inline components/organisms/ToolList/nav.tsx:7]:
```suggestion
import { setQueryParams } from "lib/utils/query-params";

import ComponentDateFilter from "components/molecules/ComponentDateFilter/component-date-filter";
import NavItem from "./nav-item";

type ToolListArray = {
```

**Comment by jpmcb** [comment]
Accepted the deploy preview and running CI/CD now.

**Review by jpmcb — APPROVED** [review]
Looks good! Thanks!!

**Review by brandonroberts — APPROVED** [review]

**Pull request merged**